tidb-cloud/features.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Features
+summary: Learn about feature support status for different TiDB Cloud plans.
+---
+
+# Features
+
+This document lists the feature support status for different TiDB Cloud plans, including {{{ .starter }}}, Essential, and Dedicated.
+
+> **Tip:**
+>
+> [TiDB Cloud Starter](/tidb-cloud/select-cluster-tier.md#starter) is the best way to get started with TiDB Cloud. Additionally, you can try out TiDB Cloud features on [TiDB Playground](https://play.tidbcloud.com/?utm_source=docs&utm_medium=tidb_cloud_quick_start).
+
+- ✅: The feature is generally available unless otherwise noted as preview.
+- ❌: The feature is currently not available.
+
+<table><thead>
+  <tr>
+    <th>Category</th>
+    <th>Feature</th>
+    <th style="text-align:center;">Starter</th>
+    <th style="text-align:center;">Essential</th>
+    <th style="text-align:center;">Dedicated</th>
+  </tr></thead>
+<tbody>
+  <tr>
+    <td>Basic</td>
+    <td>TiFlash</td>
```

Comment:
Just for the completeness, can we add a "TiKV" line above the TiFlash, and all columns are marked as checked.